ecosystem/sep-0038.md

```diff
@@ -7,9 +7,9 @@ Author: Jake Urban <@jakeurban> and Leigh McCulloch <@leighmcculloch>
 Track: Standard
 Status: Draft
 Created: 2021-04-09
-Updated: 2023-06-07
+Updated: 2023-08-24
 Discussion: https://github.com/stellar/stellar-protocol/issues/901
-Version 2.1.0
+Version 2.1.1
```

Comment:
I think this is a breaking change for those who expect alpha-3 codes, so we'll need to bump the major version and notify existing partners if we go with this approach.